### DISCUSSION TITLE
✨ Mezzanine 5.0 released ✨

### DISCUSSION THREAD

**Discussion by jerivas** [stephenmcd/mezzanine#2002] [General]
Mezzanine 5.0.0 is now available for download on PyPI:

```
pip install mezzanine==5.0.0
```

The most notable change is removal of support for Python < 3.6 and Django < 2.2. A full list of changes is available on the official GitHub [Release Notes](https://github.com/stephenmcd/mezzanine/releases/tag/v5.0.0). The package metadata now also includes up to date compatibility information for Python and Django. This information is displayed under the "Classifiers" section of the [PyPI page](https://pypi.org/project/Mezzanine/) sidebar and the project's [README](https://github.com/stephenmcd/mezzanine/tree/v5.0.0#readme).

Also notice that [filebrowser_safe 1.0](https://github.com/stephenmcd/filebrowser-safe/releases/tag/v1.0.0) and [grappelli_safe 1.0](https://github.com/stephenmcd/grappelli-safe/releases/tag/v1.0.0) have also been released and are listed as dependencies of Mezzanine 5.0.

May you enjoy this new release!

**Comment by molokov** [comment]
Thanks Ed, that's wonderful!
Are there any plans on updating Cartridge to work with Mezzanine 5.0/Django 2.2? Some good work has been done here already as a starting point: https://github.com/stephenmcd/cartridge/pull/346 but I suspect it may need further work.
I'll see what I can help with when I get some free time in the next few months.

**Reply by jerivas** [reply]
Unfortunately I don't have the bandwidth to support Cartridge right now, plus I never used it in production and I'm not familiar with it. I think we can talk to Stephen to grant write access to you or anyone else that wants to step up.

**Reply by skru** [reply]
https://github.com/stephenmcd/mezzanine/discussions/2004

I'm not sure I'd be best qualified to take over the repo, but starting to bring cartridge + third-parties up-to-date. 
But if no one else is willing I will. I have several production sites that the thought of moving to another cms is horrifying, so keeping them running is a necessity for me.

**Comment by mikenowakme** [comment]
Is there documentation on upgrading from Mezzanine 4.2 to Mezzanine 5.0? We have an old site running Django 1.8 and Mezzanine 4.2 so I assume we need to update both packages.

**Reply by jerivas** [reply]
That is correct. The general workflow is:

- Upgrade one version at a time, avoiding multi version leaps for django, mezzanine, and other packages
- For each upgrade read the release notes published by all projects
- For mezzanine make sure you copy any changes to settings and urls in your project folder from a fresh mezzanine project or the github repo
- Apply migrations
- Run your test suite
- Run the development server, which will run django's check framework and emit warnings or errors that need fixing
